packages/psss/src/views/CountryReportsView.js

```diff
@@ -23,16 +26,23 @@ const links = [
   },
 ];
 
-export const CountryReportsView = () => {
-  const { countryName } = useParams();
-  const back = {
-    url: '/',
-    title: 'Countries',
-  };
+export const CountryReportsViewComponent = ({ isMultiCountryUser }) => {
+  const { countryCode } = useParams();
+
+  let back = null;
+
+  // can this be improved
```

Comment:
I can't think of how! Maaybe would pull the logic into the mapStateToProps - I'm a big fan of really dumb presentational components

Reply:
Nice - that sound like a good improvement to me